salt/modules/ssh.py

```diff
@@ -1175,8 +1175,10 @@ def hash_known_hosts(user=None, config=None):
     if not os.path.isfile(full):
         return {'status': 'error',
                 'error': 'Known hosts file {0} does not exist'.format(full)}
+    origmode = os.stat(full).st_mode
     cmd = ['ssh-keygen', '-H', '-f', full]
     cmd_result = __salt__['cmd.run'](cmd, python_shell=False)
+    os.stat(full, origmode)
```

Comment:
No worries @folti, thanks for the ticket handle I didn't saw that. Hope it gets merged to 2016.11 soon.